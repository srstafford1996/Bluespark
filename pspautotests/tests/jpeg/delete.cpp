#include <common.h>
#include <pspjpeg.h>
#include <psputility.h>

extern "C" int main(int argc, char *argv[]) {
	sceUtilityLoadModule(PSP_MODULE_AV_AVCODEC);

	checkpointNext("Basic:");
	checkpoint("  Before init: %08x", sceJpegDeleteMJpeg());
	checkpoint("  Init: %08x", sceJpegInitMJpeg());
	checkpoint("  After init: %08x", sceJpegDeleteMJpeg());
	checkpoint("  Create: %08x", sceJpegCreateMJpeg(480, 272));
	checkpoint("  After delete: %08x", sceJpegDeleteMJpeg());
	checkpoint("  Twice: %08x", sceJpegDeleteMJpeg());

	sceUtilityUnloadModule(PSP_MODULE_AV_AVCODEC);
	return 0;
}