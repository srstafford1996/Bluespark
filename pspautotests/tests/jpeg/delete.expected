[x] Basic:
[x]   Before init: 80650039
[r]   Init: 00000000
[x]   After init: 80650039
[x]   Create: 00000000
[x]   After delete: 00000000
[x]   Twice: 80650039
